src/main/java/com/ipt/kafkatopicupdates/streams/DefaultStreamsTopology.java

```diff
@@ -0,0 +1,49 @@
+package com.ipt.kafkatopicupdates.streams;
+
+import ch.ipt.kafka.avro.Authorization;
+import org.apache.kafka.common.serialization.Serdes;
+import org.apache.kafka.streams.StreamsBuilder;
+import org.apache.kafka.streams.kstream.KStream;
+import org.apache.kafka.streams.state.KeyValueStore;
+import org.apache.kafka.streams.state.StoreBuilder;
+import org.apache.kafka.streams.state.Stores;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class DefaultStreamsTopology {
```

Comment:
IMO this is not a "default" Topology. It is a very specific Topology. e.g. AuthorizationDeduplicationTopology

Reply:
True, will change it